src/track/beats.cpp

```diff
@@ -599,12 +599,14 @@ std::optional<BeatsPointer> Beats::tryTranslate(audio::FrameDiff_t offsetFrames)
             m_markers.cend(),
             std::back_inserter(markers),
             [offsetFrames](const BeatMarker& marker) -> BeatMarker {
-                return BeatMarker(marker.position() + offsetFrames,
+                const auto position = marker.position() + offsetFrames;
```

Comment:
position -> translatedPosition?

Please add a comment that offsetFrames might be fractional and therefore we need to round after the translation. The naming "offsetFrames" sounds like we are dealing with full frames here. Why not only "offset" because of type safety? Or otherwise use "offsetInFrames" if you want to keep repeating the type in the name redundantly. The revised naming would at least reduce the chance for misunderstandings regarding the value of this variable, i.e. fractional and not full frames.

Reply:
> Please add a comment that offsetFrames might be fractional and therefore we need to round after the translation.

I could add a comment, but the `Frames` suffix is used in *a lot* of other places.

> Why not only "offset" because of type safety?

`audio::FrameDiff_t` is just a typedef for `double`, so there is no type safety :disappointed: This is why the `Frames` suffix was added in the first place.